src/modules/fundingManager/bondingCurve/FM_BC_BondingSurface_Redeeming_v1.sol

```diff
@@ -336,8 +287,13 @@ contract FM_BC_BondingSurface_Redeeming_v1 is
         ) {
             revert InvalidOrchestratorTokenWithdrawAmount();
         }
+
         token().safeTransfer(to, amount);
 
+        if (MIN_RESERVE > token().balanceOf(address(this))) {
```

Comment:
We decided to keep it here, as it creates more information in case of fail state
In case of the _Redeeming_Restricted_Repayer_Seizable_v1.sol we will adapt it though